### ISSUE TITLE
Windows and mac docker versions are not current

### ISSUE THREAD

**Issue by rfay** [drud/quicksprint#31]
Both Windows and Mac docker_installs provide docker 17.09, not the current recommended 18.03. Please update to the current version. As noted elsewhere, 
* please name the install to contain the version number
* Remove any spaces in the names (makes scripting far better and easier) 

The best place to get the docker-for-mac and docker-for-windows releases is on their release notes pages, for example https://docs.docker.com/docker-for-mac/release-notes/

**Comment by BrianGilbert** [comment]
I'm using the most current versions linked on the Docker website 
```
DOCKER_URLS="https://download.docker.com/mac/stable/23751/Docker.dmg https://download.docker.com/win/stable/16762/Docker%20for%20Windows%20Installer.exe"
```
https://docs.docker.com/docker-for-mac/release-notes/
https://docs.docker.com/docker-for-windows/release-notes/


Addressing the dot points though..

**Comment by rfay** [comment]
Yeah, 23751 (mac) is correct to download, but it not what was in the package in v0.0.5.

I do think that if the packaging script were run against an existing directory, it would have respected old docker stuff that was there, but I don't think @rickmanelius  had ever done that before last night.

**Comment by rfay** [comment]
The key point is: Build it and make sure it has the right stuff inside.

**Comment by rfay** [comment]
Also, the script shouldn't be installing docker. Make people do that, and start docker. It will never turn out right, I don't think, if the script tries to do it.

**Comment by BrianGilbert** [comment]
I know it does as the version of Docker I have installed locally was done by the script.. and it's 18.03.0-ce-mac60 (23751)

**Comment by BrianGilbert** [comment]
@Rfay, perhaps you should look at the open PR :)

**Comment by rfay** [comment]
I confirm that this is fixed for both macOS and windows in #32

**Comment by BrianGilbert** [comment]
Suspect this was a packaging error
